nixosModules/hermes/default.nix

```diff
@@ -0,0 +1,44 @@
+{
+  nix-std ? import (fetchTarball "https://github.com/chessai/nix-std/archive/master.tar.gz"),
```

Comment:
im not a huge fan of defaulting with fetchTarball. could we just make this required?

Reply:
ok. no hard need here. will remove. except ease of testing may be.